### PR TITLE
Added grid-supporting inverter

### PR DESCRIPTION
**Pull request by SabineAuer**
such as used for connection of PV power plants. This inverter has over-frequency control above a certain threshold.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -16,6 +18,7 @@ NLsolve = "2774e3e8-f4cf-5e23-947b-6d7e65073b56"
 NetworkDynamics = "22e9dc34-2a0d-11e9-0de0-8588d035468b"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```
Comment (janlisse) [inline Project.toml]: Can you remove this dependency please?

File: Project.toml [hunk 2]
```diff
@@ -4,7 +4,9 @@ authors = ["Tim Kittel", "Jan Liße", "Sabine Auer"]
 version = "2.2.0"
 
 [deps]
+CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
```
Comment (janlisse) [inline Project.toml]: Why is this new dependency needed?

File: Project.toml [hunk 3]
```diff
@@ -4,7 +4,9 @@ authors = ["Tim Kittel", "Jan Liße", "Sabine Auer"]
...
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
```
Comment (janlisse) [inline Project.toml]: Same as above

File: Project.toml (hunk 2 above)
Comment (janlisse) [inline Project.toml]: Why is this new dependency needed?

File: Project.toml (hunk 3 above)
Comment (janlisse) [inline Project.toml]: Same as above

File: Project.toml (hunk 1 above)
Comment (janlisse) [inline Project.toml]: Can you remove this dependency please?

File: Project.toml (hunk 2 above)
Comment (SabineAuer) [reply Project.toml]: I forgot to remove all the unnecessary dependencies you mentioned and made up for this now in this [commit](https://github.com/JuliaEnergy/PowerDynamics.jl/pull/55/commits/b1e67c4e0c5b4f7b8dd9c0648ba1db0344de99a8)

**Pull request merged**